ui/src/components/executions/ExecutionRoot.vue

```diff
@@ -79,6 +86,11 @@
             },
         },
         methods: {
+            downloadExecution() {
+                const blob = new Blob([JSON.stringify(this.execution)], {type: "application/octet-stream"});
+                const url = window.URL.createObjectURL(blob)
+                Utils.downloadUrl(url, `execution_${this.$route.params.id}.txt`);
```

Comment:
Put it in a json file as it's json content and format it if possible for more readability